### PR TITLE
Add serial and parallel satellite scanning example.

### PR DESCRIPTION
**Pull request by tskisner**
This creates separate notebooks for the serial and parallel satellite scanning example.

### CODE REVIEW

**Comment by zonca** [comment]
I see `simscan.ipynb` still in the folder, should that be removed?

File: lessons/02_Simulated_Scan_Strategies/simscan_satellite_mpi.ipynb [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    "\n",
+    "# Load common tools for all lessons\n",
+    "import sys\n",
+    "sys.path.insert(0, \"..\")\n",
+    "from lesson_tools import (\n",
+    "    check_nersc,\n",
+    ")\n",
+    "nersc_host, nersc_repo = check_nersc(reservation=nersc_reservation)\n",
+    "if nersc_host is not None:\n",
+    "    %reload_ext slurm_magic"
```
Comment (zonca) [inline lessons/02_Simulated_Scan_Strategies/simscan_satellite_mpi.ipynb:28]: I get `ModuleNotFoundError: No module named 'slurm_magic'.`, I am using `CMB 20191008`, do we need to install the slurm magic in the CMBENV environment?

**Review by zonca — COMMENTED** [review]
coverage is very good, I'll use this for my sky model lesson

**Review by zonca — APPROVED** [review]
once the slurm issue is figured out, you can go ahead an merge them

**Comment by tskisner** [comment]
Ok, I the slurm-magic package is already in cmbenv, but needs to be updated at NERSC.  As for the simscan.ipynb, I left that original file since I believe Reijo has a huge pile of changes that have not been pushed and I did not want to touch the original file to cause a merge conflict.  Presumably his future commit will do a rename of that file to the ground scan one.

**Comment by tskisner** [comment]
@zonca, something has changed since the system came back online.  My previously installed cmbenv software stack now gives me this when loading:
```
$ module use /global/common/software/cmb/cori/default/modulefiles
$ module load cmbenv
cmbenv/20191008(19):ERROR:150: Module 'cmbenv/20191008' conflicts with the currently loaded module(s) 'python/3.6-anaconda-4.4'
cmbenv/20191008(19):ERROR:102: Tcl command execution failed: conflict cmbenv python
```
It looks like Rollin is now loading a default python version at boot.  I'll contact him.  Note that my kernel **does** have nbstripout, slurm-magic, etc installed, but the kernel hits that error and never loads.

**Comment by tskisner** [comment]
The module error above was actually my fault (stale testing code in my shell resource file).  It looks like nbstripout made it into the last cmbenv install, but not slurm-magic.  I'll leave this issue open until I update that install.

**Comment by tskisner** [comment]
Ok I just installed version 20191012 and made it the default.  After loading it and installing the kernel, you may have to restart your session before it becomes available.  I think all the changes are in there.  I'll fix this conflict and then merge.

**Pull request merged**